diff -up ./configure.in.orig ./configure.in
--- ./configure.in.orig	2018-05-13 08:59:07.547223413 +0300
+++ ./configure.in	2018-05-13 08:59:24.106274751 +0300
@@ -12,7 +12,6 @@ AM_INIT_AUTOMAKE(rootcerts, 0.50)
 
 EXTRA_PATH=$PATH:/sbin:/usr/sbin:/usr/local/sbin:/usr/local/bin
 AC_PATH_PROG(OPENSSL, openssl, "", $EXTRA_PATH)
-AC_PATH_PROG(REHASH, c_rehash, "", $EXTRA_PATH)
 AC_PATH_PROGS(PERL, perl5 perl, "", $EXTRA_PATH)
 
 if test "$PERL" = ""
@@ -46,20 +46,6 @@ echo "dir=\"$certdb\"" >rootcertsdir.cnf
 AC_SUBST(certdb)
 AC_SUBST(OPENSSL)
 
-if test "$OPENSSL" != ""
-then
-	if test "$REHASH" = ""
-	then
-		AC_MSG_WARN(Unable to locate OpenSSL's c_rehash script in the current)
-		AC_MSG_WARN(PATH.  The c_rehash script is included in the OpenSSL)
-		AC_MSG_WARN(package but may not be installed on your system.  Please)
-		AC_MSG_WARN(install the c_rehash script from OpenSSL and rerun this)
-		AC_MSG_WARN(configure script.)
-		AC_MSG_ERROR(openssl found but c_rehash was not found.)
-	fi
-fi
-AC_SUBST(c_rehash)
-
 AM_CONDITIONAL(HAVE_OPENSSL, test "$OPENSSL" != "")
 
 AC_OUTPUT(Makefile)
diff -up ./Makefile.am.orig ./Makefile.am
--- ./Makefile.am.orig	2018-05-13 09:00:52.069546806 +0300
+++ ./Makefile.am	2018-05-13 09:01:09.424600369 +0300
@@ -19,7 +19,7 @@ rootcerts.cnf: config.status certs.sh
 	rm -rf certs
 	mkdir certs
 	@SHELL@ $(srcdir)/certs.sh
-	@REHASH@ certs >/dev/null
+	@OPENSSL@ rehash -v certs
 	echo 'files="'`ls certs`'"' >rootcerts.cnf
 
 install-data-hook: rootcerts.cnf
